adding-a-new-module/template/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+plugins {
+    kotlin("multiplatform")
+    id("jacoco")
+    id("maven-publish")
+    id("com.android.library")
+    id("org.jlleitschuh.gradle.ktlint")
+}
+
+val ext = (gradle as ExtensionAware).extra
+
+apply(from = "../gradle/publishable_component.gradle")
+
+group = "com.splendo.kaluga"
+version = ext["library_version"]!!
+
+repositories {
+    google()
+    mavenCentral()
+}
+
+dependencies {
+    /* Uncomment this line if you are using mockito
+     testImplementation("org.mockito:mockito-core:2.28.2")
```

Comment:
Hardcoded version (if something will be shared by multiple modules or with example project we want to use a var), but also not needed I assume? 

`android_dependencies.gradle` has `org.mockito:mockito-core:3.1.0'` (note that since it is the only place this is declared, no var is needed per se)